cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/upscale/AzureUpscaleService.java

```diff
@@ -77,9 +85,14 @@
             Deployment templateDeployment =
                     azureTemplateDeploymentService.getTemplateDeployment(client, stack, ac, azureStackView, AzureInstanceTemplateOperation.UPSCALE);
             LOGGER.info("Created template deployment for upscale: {}", templateDeployment.exportTemplate().template());
-            List<CloudResource> newInstances = azureUtils.getInstanceCloudResources(cloudContext, templateDeployment, scaledGroups);
+
+            List<CloudResource> cloudResources = azureCloudResourceService.getCloudResources(templateDeployment);
+            cloudResources.forEach(cloudResource -> resourceNotifier.notifyAllocation(cloudResource, cloudContext));
```

Comment:
There is a method for this in azureCloudResourceService that seems to be the equivalent of this.